src/fheroes2/dialog/dialog_armyinfo.cpp

```diff
@@ -610,20 +610,17 @@ void DrawMonster( fheroes2::RandomMonsterAnimation & monsterAnimation, const Tro
         monsterAnimation.increment();
 }
 
-int Dialog::ArmyJoinFree( const Troop & troop, Heroes & hero )
+int Dialog::ArmyJoinFree( const fheroes2::Text & dialog_title, const fheroes2::Text & message, const Troop & troop, Heroes & hero )
 {
     fheroes2::Display & display = fheroes2::Display::instance();
     const bool isEvilInterface = Settings::Get().ExtGameEvilInterface();
 
     // setup cursor
     const CursorRestorer cursorRestorer( true, Cursor::POINTER );
 
-    const Text title( _( "Followers" ), Font::YELLOW_BIG );
+    const Text title( dialog_title.text(), Font::YELLOW_BIG );
```

Comment:
I did a bad job explaining but this is where I wanted to use `fheroes2::Text` instead of the old `Text` class. It would be something like this:
```suggestion
    const fheroes2::Text title( dialogTitle, fheroes2::FontType::normalYellow() );
```